### PR TITLE
Create fixQ plan

### LINKED ISSUE #92: Add fixQ energy scan

### PR DESCRIPTION
**Pull request by gfabbris**
Adds "fixq" option to all plans that will keep the (HKL) constant during an energy scan (or any scan really...).

Closes #92.

### CODE REVIEW

**Review by strempfer — APPROVED** [review]
Great to have qxscan and fixq possibility in one scan.
>  _Particularly used for energy scan._ 
Sentence perhaps not needed here since it is an energy scan anyway.

File: profile_bluesky/startup/instrument/plans/local_scans.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def qxscan(edge_energy, time=None, detectors=None, lockin=False, dichro=False,
         point using the +, -, -, + sequence, thus increasing the number of
         points by a factor of 4
     fixq : boolean, optional
-        Flag to fix the hkl position. Particularly used for energy scan.
+        Flag to fix the hkl position.
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/plans/local_scans.py:308]: True=?, False=?

File: profile_bluesky/startup/instrument/devices/huber.py [hunk 2]
```diff
@@ -70,6 +69,16 @@ class FourCircleDiffractometer(E4CV):
     energy_update_calc_flag = Component(Signal, value=1)
     energy_offset = Component(Signal, value=0)
 
+    # TODO: This is needed to prevent busy plotting.
+    @property
+    def hints(self):
+        fields = []
+        for _, component in self._get_components_of_kind(Kind.hinted):
+            if (~Kind.normal & Kind.hinted) & component.kind:
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/devices/huber.py:77]: Did not know about the bitwise inverse operator `~`.  Hrmph!

**Review by prjemian — APPROVED** [review]
documentation adjustments as noted - Good job!

File: profile_bluesky/startup/instrument/plans/local_scans.py [hunk 3]
```diff
@@ -141,11 +141,13 @@ def lup(*args, time=None, detectors=None, lockin=False, dichro=False,
...
     fixq : boolean, optional
-        Flag to fix the hkl position. Particularly used for energy scan.
+        Flag for fixQ scans. If True, it will fix the diffractometer hkl
+        position during the scan. This is particularly useful for energy scan.
+        Note that hkl is moved ~after~ the other motors!
     md : dictionary, optional
         Metadata to be added to the run start.
 
-    See Also
+    See Alsos
```
Comment (prjemian) [inline profile_bluesky/startup/instrument/plans/local_scans.py:150]: Alsos?

File: profile_bluesky/startup/instrument/devices/huber.py (hunk 2 above)
Comment (gfabbris) [reply profile_bluesky/startup/instrument/devices/huber.py:77]: Copied from `ophyd.signal.Signal` 😅

https://github.com/bluesky/ophyd/blob/7937cd46e92e9ec2f7eec321dd5b1b1b71115e20/ophyd/signal.py#L457

**Review by prjemian — APPROVED** [review]

**Pull request merged**